bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/api/Car.java

```diff
@@ -128,80 +196,114 @@ public boolean isDisableLocation() {
         return disableLocation;
     }
 
-    public void setDisableLocation(boolean disableLocation) {
-        this.disableLocation = disableLocation;
-    }
-
     public boolean isDisableBattery() {
         return disableBattery;
     }
 
-    public void setDisableBattery(boolean disableBattery) {
-        this.disableBattery = disableBattery;
-    }
-
     public boolean isDisableCockpit() {
         return disableCockpit;
     }
 
-    public void setDisableCockpit(boolean disableCockpit) {
-        this.disableCockpit = disableCockpit;
-    }
-
     public boolean isDisableHvac() {
         return disableHvac;
     }
 
-    public void setDisableHvac(boolean disableHvac) {
-        this.disableHvac = disableHvac;
-    }
-
     public @Nullable Double getBatteryLevel() {
         return batteryLevel;
     }
 
-    public void setBatteryLevel(Double batteryLevel) {
-        this.batteryLevel = batteryLevel;
-    }
-
     public @Nullable Boolean getHvacstatus() {
         return hvacstatus;
     }
 
-    public void setHvacstatus(Boolean hvacstatus) {
-        this.hvacstatus = hvacstatus;
-    }
-
     public @Nullable Double getOdometer() {
         return odometer;
     }
 
-    public void setOdometer(Double odometer) {
-        this.odometer = odometer;
-    }
-
     public @Nullable String getImageURL() {
         return imageURL;
     }
 
-    public void setImageURL(String imageURL) {
-        this.imageURL = imageURL;
-    }
-
     public @Nullable Double getGpsLatitude() {
         return gpsLatitude;
     }
 
-    public void setGpsLatitude(Double gpsLatitude) {
-        this.gpsLatitude = gpsLatitude;
-    }
-
     public @Nullable Double getGpsLongitude() {
         return gpsLongitude;
     }
 
-    public void setGpsLongitude(Double gpsLongitude) {
-        this.gpsLongitude = gpsLongitude;
+    public @Nullable String getLocationUpdated() {
+        return locationUpdated;
+    }
+
+    public @Nullable Double getExternalTemperature() {
+        return externalTemperature;
+    }
+
+    public @Nullable Double getEstimatedRange() {
+        return estimatedRange;
+    }
+
+    public PlugStatus getPlugStatus() {
+        return plugStatus;
+    }
+
+    public ChargingStatus getChargingStatus() {
+        return chargingStatus;
+    }
+
+    public ChargingMode getChargingMode() {
+        return chargingMode;
+    }
+
+    public @Nullable Integer getChargingRemainingTime() {
+        return chargingRemainingTime;
+    }
+
+    public @Nullable Double getBatteryAvailableEnergy() {
+        return batteryAvailableEnergy;
+    }
+
+    public Double getHvacTargetTemperature() {
+        return hvacTargetTemperature;
+    }
+
+    public void setHvacTargetTemperature(Double hvacTargetTemperature) {
```

Comment:
Parameter could be a simple `double`. Make sense as the class member is now a `double`.